### PR TITLE
Fix styles of Recaptcha::ClientHelper

### PR DESCRIPTION
**Pull request by ydakuka**
https://github.com/ambethia/recaptcha/issues/278

Fixed all errors except `scrolling="no"`.

### CODE REVIEW

File: lib/recaptcha/client_helper.rb [hunk 1]
```diff
@@ -25,9 +25,9 @@ def recaptcha_tags(options = {})
                 <div style="width: 302px; height: 422px; position: absolute;">
                   <iframe
                     src="#{fallback_uri}"
-                    frameborder="0" scrolling="no"
-                    style="width: 302px; height:422px; border-style: none;">
-                    title="ReCAPTCHA"
+                    scrolling="no"
+                    style="width: 302px; height: 422px; border-style: none; border: 0;">
+                    ReCAPTCHA
```
Comment (grosser) [inline lib/recaptcha/client_helper.rb]: that has to stay the title ?

**Comment by grosser** [comment]
would be great if we had tests for this too, but I guess that's pretty hard ... 
the title change looks wrong though, the error was that there is text in the iframe and now there is text again ...

**Comment by grosser** [comment]
can you also explain how you got these errors for anyone wanting to change/reproduce this later

**Comment by ydakuka** [comment]
I got these errors by analyzing the page with the recaptcha through the site https://validator.w3.org/

File: lib/recaptcha/client_helper.rb (hunk 1 above)
Comment (ydakuka) [reply lib/recaptcha/client_helper.rb]: I moved the title to the name attribute. It's ok?

**Comment by grosser** [comment]
awesome!

**Comment by grosser** [comment]
4.8.1

might also be fun to use https://validator.w3.org/ as an api and pass in generated html as a test

**Pull request merged**